java/ql/src/semmle/code/java/frameworks/play/Play.qll

```diff
@@ -0,0 +1,167 @@
+import java
+
+/**
+ * Play MVC Framework Result Class
+ */
+class PlayMVCResultClass extends Class {
+  PlayMVCResultClass() { this.hasQualifiedName("play.mvc", "Result") }
+}
+
+/**
+ * Play MVC Framework Results Class
+ *
+ * Documentation: https://www.playframework.com/documentation/2.8.x/JavaActions
+ */
+class PlayMVCResultsClass extends Class {
+  PlayMVCResultsClass() { this.hasQualifiedName("play.mvc", "Results") }
+}
+
+/**
+ * Play MVC Framework HTTP Request Header Class
+ */
+class PlayMVCHTTPRequestHeader extends RefType {
```

Comment:
This is a great example of why we should use camelCase/PascalCase for abbreviations in concatenated identifiers: `PlayMvcHttpRequestHeader` is much more readable.